controllers/validator.go

```diff
@@ -95,7 +92,7 @@ func (r *DPAReconciler) ValidateVeleroPlugins(log logr.Logger) (bool, error) {
 		return false, err
 	}
```

Comment:
missing removal of get call here
https://github.com/openshift/oadp-operator/blob/426fd7035609cb820714eee1ef1c9e4b31bf2f7d/controllers/validator.go#L93-L96

Reply:
fixed